examples/sampling/graphbolt/node_classification.py

```diff
@@ -92,6 +92,18 @@ def create_dataloader(
 
     ############################################################################
     # [Step-2]:
+    # self.copy_to()
+    # [Input]:
+    # 'device': The device to copy the data to.
+    # 'extra_attrs': The extra attributes to copy.
+    # [Output]:
+    # A CopyTo object to copy the data to the specified device.
+    ############################################################################
+    if args.storage_device != "cpu":
+        datapipe = datapipe.copy_to(device=device, extra_attrs=["seed_nodes"])
```

Comment:
Can you add a comment to clarify this is for using CUDA sampling?

Also I am wondering whether we need a CUDA version of item sampler as well, such that we can move all seed nodes to CUDA, and shuffle from there.

Reply:
I actually talked to @Rhett-Ying about it some time ago. However, since we wrap with a Prefetcher before the CopyTo in the dataloader, it actually doesn't cause a performance issue as the itemsampler runs on a separate thread with buffering.